### ISSUE TITLE
Enhancement request: use info bar instead of popup for errors

### ISSUE THREAD

**Issue by smithfred** [gnome-mpv/gnome-mpv#308]
Having a modal error window means the user has to dismiss it before doing anything else (e.g. closing GNOME MPV).

An info bar (https://developer.gnome.org/hig/stable/info-bars.html.en) would provide the same info but without blocking other actions (i.e. one click fewer).

**Comment by TingPing** [comment]
The better design would be: https://developer.gnome.org/hig/stable/in-app-notifications.html.en

An implementation of it is in libgd (a subproject library, so not an extra dep).

**Comment by smithfred** [comment]
Yes, I thought of suggesting that, but I didn't because it's not a GTK widget (!) so would be extra work.

But if it's as easy then it would achieve the same result.
